src/_cffi_src/build_commoncrypto.py

```diff
@@ -0,0 +1,32 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import absolute_import, division, print_function
+
+from _cffi_src.utils import build_ffi_for_binding
+
+
+_module_prefix = "_cffi_src.commoncrypto."
+_modules = [
+    "cf",
+    "common_digest",
+    "common_hmac",
+    "common_key_derivation",
+    "common_cryptor",
+    "common_symmetric_key_wrap",
+    "secimport",
+    "secitem",
+    "seckey",
+    "seckeychain",
+    "sectransform",
+]
```

Comment:
This is a bit silly, but can you move these values into the `build_ffi_for_binding` call so it looks a bit more declaraitive, and less imperative.